service/src/main/java/bio/terra/policy/service/pao/PaoService.java

```diff
@@ -326,18 +326,13 @@ private List<PolicyConflict> mergeAttributes(Pao sourcePao, Pao destinationPao)
 
     for (PolicyInput input : sourcePao.getAttributes().getInputs().values()) {
       PolicyInput destinationMatchedPolicy = policyInputs.lookupPolicy(input);
-      // If the policy does not exist in the destination, so we add it
-      if (destinationMatchedPolicy == null) {
-        policyInputs.addInput(input);
+      PolicyInput resultInput = PolicyMutator.combine(destinationMatchedPolicy, input);
```

Comment:
This was a short-cut to skip combine, but you need to do combine so you can enforce specific rules for group, right?

Reply:
yep, that's right.